spec/domain/importers/content_details_spec.rb

```diff
@@ -2,14 +2,21 @@
   let(:base_path) { '/base_path' }
   let(:content_id) { 'content_id' }
 
-  subject { Importers::ContentDetails.new(content_id, base_path) }
+
 
   context 'Import contents' do
-    let(:existing_content_hash) { 'ContentHashContentHash' }
-    let(:parsed_content_hash) { 'NewContentHash' }
+    subject { Importers::ContentDetails.new(latest_dimension_item.id) }
+    let(:locale) { 'en' }
+    let!(:existing_content_hash) { 'ContentHashContentHash' }
+    let!(:parsed_content_hash) { 'NewContentHash' }
+    let!(:latest_dimension_item_fr) do
+      create(:dimensions_item,
+        content_id: content_id, base_path: base_path, locale: 'fr',
+        latest: true, raw_json: nil, content_hash: existing_content_hash)
+    end
     let!(:latest_dimension_item) do
       create(:dimensions_item,
-        content_id: content_id, base_path: base_path,
+        content_id: content_id, base_path: base_path, locale: 'en',
```

Comment:
Shouldn't `en` be set automatically for all DimensionItems in the `FactoryBot` factory?